lib/plugins/aws/index.js

```diff
@@ -33,7 +33,7 @@ class SDK {
 
   request(service, method, params, stage, region) {
```

Comment:
But I have this on my todo list to create a task for this and for how we handle regions. Because the region handling at the moment is super weird (and e.g. default regions in profiles are not used)